packages/beachball/src/__e2e__/monorepo/getScopedPackages.test.ts

```diff
@@ -38,7 +38,7 @@ describe('getScopedPackages', () => {
     expect(scopedPackages.includes('bar')).toBeTruthy();
   });
 
-  fit('can mix and match with excluded packages', async () => {
+  it('can mix and match with excluded packages', async () => {
```

Comment:
oops! should probably add a lint for this thing... isn't the first time my debug stuff got leaked in here.